src/main/resources/scalastyle_definition.xml

```diff
@@ -1,220 +1,220 @@
-<?xml version="1.0" encoding="UTF-8"?>
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
Aren't you precisely removing that whitespace from all other lines?

Reply:
VSCode seems to format it automatically like this, perhaps that's a convention in XML?

Reply:
Not sure, but it feels weird. Anyways, if it is automatically formatted there isn't much to do.

Reply:
I removed it as it does feel weird =]